packages/puppeteer-core/src/api/Page.ts

```diff
@@ -345,6 +353,42 @@ export interface ScreencastOptions {
    * @defaultValue `1`
    */
   speed?: number;
+  /**
+   * Specifies the frame rate in frames per second.
+   *
+   * @defaultValue `30` (`20` for GIF)
+   */
+  fps?: number;
+  /**
+   * Specifies the number of times to loop playback, from `0` to `Infinity`.
+   *
+   * @defaultValue `0`
```

Comment:
I think we want a little better explanation here, 
As if undefined is provided no loop;
If 0 or Infinity is provided - loop Infinity
And default is undefined.